src/rules/selector-no-redundant-nesting-selector/README.md

```diff
@@ -59,3 +59,57 @@ p {
   }
 }
 ```
+
+## Options
+
+`string[]`: `[keywords]`
```

Comment:
Sorry for nitpicking, but this should be `ignoreKeywords: ["/regex/", /regex/, "string"]`. The rule could be updated to also support regex values.

Here's an example:
https://github.com/stylelint/stylelint/tree/master/lib/rules/font-family-no-duplicate-names#ignorefontfamilynames-regex-regex-string

Reply:
This is the right pick，I will update immediately, remember to wait for me